ntbs-service/Pages/Notifications/Edit/ClinicalDetails.cshtml

```diff
@@ -443,15 +443,15 @@
             <nhs-form-group nhs-form-group-type="@ShortCourseGroupState" v-bind:class="{ 'nhsuk-form-group--error': hasError}">
                 <nhs-fieldset>
                     <nhs-fieldset-legend nhs-legend-size="Standard">Has Started Short Course Treatment?</nhs-fieldset-legend>
-                    <span ref="errorField[0]" nhs-span-type="ErrorMessage" asp-validation-for="ClinicalDetails.IsShortCourseTreatment"></span>
+                    <span ref="errorField[0]" nhs-span-type="ErrorMessage" asp-validation-for="ClinicalDetails.IsShortCourseTreatment" id="short-course-error"></span>
                     <nhs-radios nhs-radios-type="Standard">
                         <nhs-radios-item>
-                            <input nhs-input-type="Radios" asp-for="ClinicalDetails.IsShortCourseTreatment" id="short-course-yes" type="radio" value="true" ref="input[0]" v-on:change="validate" />
-                            <label nhs-label-type="Radios" for="short-course-yes">Yes</label>
+                            <input nhs-input-type="Radios" asp-for="ClinicalDetails.IsShortCourseTreatment" type="radio" value="true" ref="input[0]" v-on:change="validate" />
+                            <label nhs-label-type="Radios" asp-for="ClinicalDetails.IsShortCourseTreatment">Yes</label>
```

Comment:
I'm worried this produces identical ids for the two, which breaks accessibility (as labels aren't linked properly).
`asp-for` is still useful on the inputs (which gain `name` from it), but afraid we need to provide an `id` attribute in addition by hand

Reply:
Ah so this was the only radio/checkbox field (at least on this page, but possible elsehwere too) where we manually specified an id, but you're obviously right that where there are multiple asp-for bindings for the same field there will be duplicate id fields.

Should I fix this everywhere as part of this ticket, or should we raise a tech debt ticket?

Reply:
If you can fix it on this page, that'd be great. Other places I've been keeping an eye out for so it should be good (🤞). if you spot any as you work, then definitely worth fixing - not sure it's worht a whole debt ticket really :)